src/Network/NWFramer.cs

```diff
@@ -24,8 +24,8 @@
 namespace Network {
 
 	[TV (13,0), Mac (10,15), iOS (13,0), Watch (6,0)]
-	public class NWFramer : NWProtocolMetadata {
-		public NWFramer (IntPtr handle, bool owns) : base (handle, owns) {}
+	public class NWFramer : NativeObject /* : NWProtocolMetadata */ {
```

Comment:
why comment ?

Reply:
@spouliot for me to remember, but I can remove it

Reply:
it's not about removal - it's about _why_ ? :)
if it's a subclass of `NWProtocolMetadata` then why not make it so today ?
if not then why keep it as a comment ?